src/silx/gui/widgets/FrameBrowser.py

```diff
@@ -294,13 +329,17 @@ def __init__(self, parent=None):
         self._playButton.setToolTip("Display movie with frames")
         self._playButton.setIcon(icons.getQIcon("camera"))
         self._playButton.setIconSize(iconSize)
+        self._playButton.setCheckable(True)
         self.mainLayout.addWidget(self._playButton)
         
-        self._playButton.clicked.connect(self._playStopSequence)
+        self._playButton.toggled.connect(self._playButtonToggled)
         self._menuPlaySlider = _PlayButtonContextMenu(self)
+        self._menuPlaySlider.sigFrameRateChanged.connect(self._frameRateChanged)
```

Comment:
Update the timer interval when changing the frame rate: This enables changing the frame rate while playing